database/migrations/2016_09_01_125807_add_name_location_url_to_friends.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
```

Comment:
The style on this project is to alphabetize the `use` block instead of ordering by length. Would you alpha this please?